Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -290,12 +290,23 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         // We might have some non-fatal output (warnings/notes) from the compiler even when
         // we were able to parse the manifest successfully.
         if let compilerOutput = result.compilerOutput {
+            // FIXME: Isn't the strategy too naive?
```

Comment:
Perhaps expand a bit on this - what would be a less naive strategy to employee here?  I'm not sure I have any ideas off the top of my head which would address the FIXME.

Reply:
Done